src/tools/perf/lib/ucp_tests.cc

```diff
@@ -283,8 +289,9 @@ class ucp_perf_test_runner {
     send(ucp_ep_h ep, void *buffer, unsigned length, ucp_datatype_t datatype,
          uint8_t sn, uint64_t remote_addr, ucp_rkey_h rkey)
     {
+        ucp_request_param_t param = {};
```

Comment:
it should be faster to make "param" a member variable than allocate it on stack - less cache misses